contracts/mocks/MockRedemptionManager.sol

```diff
@@ -35,10 +35,11 @@ contract MockRedemptionManager is TrackedRedemption {
      * @param _value The number of tokens to redeem
      */
     function redeemTokenByOwner(uint256 _value) public {
-        require(tokenToRedeem[msg.sender] >= _value);
+        require(tokenToRedeem[msg.sender] >= _value, "Value is greater than available");
```

Comment:
```suggestion
        require(tokenToRedeem[msg.sender] >= _value, "Insufficient tokens redeemable");
```

The error statements are printed when the require statement fails and reverts. In this case, the statement will be printed when `tokenToRedeem[msg.sender]` is LESS than `_value`. So the error statement must mention what exactly went wrong.